### PR TITLE
Ma 955/hybrid mobile apps documentation update

### PR DESCRIPTION
**Pull request by PurushYM**
For JS frameworks IDE is automatically removing quotes for key in payload JSON

please quotes on key for

- Cordova
- Reaft-native

### CODE REVIEW

File: docs/platform_concepts/channelConfiguration/cordova.md [hunk 1]
```diff
@@ -74,6 +92,14 @@ For passing data from bot to app refer bot [Bot Events](#bot-events)
 Payload is securely passed in HTTPS post request to protect the information passed in it
 :::
 
+#### Trigger journey
+
+A specific journey can be triggered on launch, by passing the slug in the payload.
+
+```js
+cordova.plugins.ymchat.setPayload({ JourneySlug: "my-journey-slug" });
```
Comment (PurushYM) [inline docs/platform_concepts/channelConfiguration/cordova.md:100]: Ignore the quotes here IDE is automatically removing the quotes for key

File: docs/platform_concepts/channelConfiguration/react-native.md [hunk 2]
```diff
@@ -112,6 +127,14 @@ For passing data from bot to app refer bot [Bot Events](#bot-events)
 Payload is securely passed in HTTPS post request to protect the information passed in it
 :::
 
+#### Trigger journey
+
+A specific journey can be triggered on launch, by passing the slug in the payload.
+
+```js
+YmChat.setPayload({ JourneySlug: "my-journey-slug" });
```
Comment (PurushYM) [inline docs/platform_concepts/channelConfiguration/react-native.md:135]: Ignore the quotes here IDE is automatically removing the quotes for key

File: docs/platform_concepts/channelConfiguration/react-native.md [hunk 3]
```diff
@@ -101,7 +116,7 @@ Note: Firebase service account key is required to send notifications. You can sh
 Additional information can be passed in the form of key value pair from app to bot using payload.
 
 ```javascript
-YMChat.setPayload({ name: "Purush", age: 21 });
+YMChat.setPayload({ name: "Integration", type: "react-native" });
```
Comment (PurushYM) [inline docs/platform_concepts/channelConfiguration/react-native.md:119]: Ignore the quotes here IDE is automatically removing the quotes for key

File: docs/platform_concepts/channelConfiguration/cordova.md [hunk 4]
```diff
@@ -63,7 +78,10 @@ Note: Firebase service account key is required to send notifications. You can sh
 Additional information can be passed in the form of key value pair from app to bot using payload.
 
 ```javascript
-cordova.plugins.ymchat.setPayload({ name: "yellow" });
+cordova.plugins.ymchat.setPayload({
```
Comment (PurushYM) [inline docs/platform_concepts/channelConfiguration/cordova.md:81]: Ignore the quotes here IDE is automatically removing the quotes for key

**Review by sukrit-ym — APPROVED** [review]

File: docs/platform_concepts/channelConfiguration/android.md [hunk 5]
```diff
@@ -150,6 +151,17 @@ For passing data from bot to app refer bot [Bot Events](#bot-events)
...
 :::
 
+#### Trigger journey
+
+A specific journey can be triggered on launch, by passing the slug in the payload.
+
+```java
+HashMap<String, Object> payloadData = new HashMap<>();
+//Setting Payload Data
+payloadData.put("JourneySlug", "my-journey-slug");
```
Comment (kaunteya) [inline docs/platform_concepts/channelConfiguration/android.md:161]: use a real life exmaple like "login" or "checkout-cart"

**Review by kaunteya — APPROVED** [review]

**Review by ashroMithril — APPROVED** [review]

**Comment by ashroMithril** [comment]
looks good

**Pull request merged**